index.html

```diff
@@ -58,6 +59,213 @@ <h1 class="identity_text">I'm Simba, I'm glad to see you!</h1>
+        </li>
+        <li class="technology">
+          <p class="technology_text">css</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">javascript</p>
+        </li>
+      </ul>
+      <button class="project_button" type="submit">See Project</button>
+    </div>
+
+    <div class="project">
+      <img src="Snapshoot_Portfolio3.png" class="project_img" alt="project_image">
+      <h1 class="project_title">Multi-Post Stories</h1>
+      <ul class="project_details">
+        <li class="list_items">
+          <p class="list_items_text">CANOPY</p>
+        </li>
+        <li class="list_items">
+          <img src="Counter.png" alt="counter" class="counter_class">
+        </li>
+        <li class="list_items">
+          <p class="list_items_text_other">Back End Dev</p>
+        </li>
+        <li class="list_items">
+          <img src="Counter.png" alt="counter" class="counter_class">
+        </li>
+        <li class="list_items">
+          <p class="list_items_text_other">2015</p>
+        </li>
+      </ul>
+      <p class="project_description">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+      <ul class="technology_used">
+        <li class="technology">
+          <p class="technology_text">html</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">css</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">javascript</p>
+        </li>
+      </ul>
+      <button class="project_button" type="submit">See Project</button>
+    </div>
+  </section>
+
+  <section class="aboutme_container">
+    <div class="aboutme_class">
+      <h1 class="identity_text">About Myself</h1>
+      <p class="description_text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <p class="connect_text">LET'S CONNECT</p>
+      
+      <ul class="social_media_icons">
+        <li>
+          <a>
+          <img src="twitter.png" class="icons" alt="twitter">
+          </a>
+        </li>
+        <li>
+          <a>
+          <img src="github.png" class="icons" alt="github">
+        </a>
+        </li>
+        <li>
+          <a>
+          <img src="Linkedin_icon.png" class="icons" alt="linkedin">
+          </a>
+        </li>
+        <li>
+          <a>
+          <img src="medium.png" class="icons" alt="medium">
+          </a>
+        </li>
+        <li>
+          <a>
+          <img src="angellist.png" class="icons" alt="angellist">
+          </a>
+        </li>
+      </ul>
+      <button class="resume_button" type="submit">Get my resume</button>
+      <ul class="technology_and_skills">
+        <li class="technology_and_skills_item">
+          <h1 class="technology_and_skills_text">Languages</h1>
```

Comment:
- [ ] Please fix the padding in the buttons 
  
![image](https://user-images.githubusercontent.com/74744787/159058671-83c518cb-e9ca-4318-b7f8-36a5fd4a6850.png)

Figma design : 

![image](https://user-images.githubusercontent.com/74744787/159058730-601e7541-f733-4231-bf4f-78508e7b50f4.png)